include/swift/AST/DiagnosticsSema.def

```diff
@@ -2833,9 +2833,9 @@ NOTE(missing_member_type_conformance_prevents_synthesis, none,
      "protocol %2, preventing synthesized conformance "
      "of %3 to %2",
      (unsigned, Type, Type, Type))
-NOTE(classes_automatic_protocol_synthesis,none,
-    "automatic synthesis of '%0' is not supported for classes",
-    (StringRef))
+NOTE(automatic_protocol_synthesis_unsupported,none,
+    "automatic synthesis of '%0' is not supported for %select{classes|structs}1",
+    (StringRef, unsigned))
```

Comment:
Please make this modification in `include/swift/AST/diagnostics/en.yaml` as well, that would help us a lot :)

Reply:
Done :)

Reply:
Awesome, thank you!